core/modules/comment/comment.module

```diff
@@ -262,6 +249,18 @@ function comment_menu_alter(&$items) {
   $items['admin/structure/types/manage/%comment_menu_node_type/comment/display']['weight'] = 4;
 }
 
+/**
+ * Implements hook_url_outbound_alter().
+ */
+function comment_url_outbound_alter(&$path, &$options, $original_path) {
+  // Include a security token in the query if approving from the
+  // manage comments pages. Only if it doesn't already have a token.
+  if (preg_match('/comment\/[1-9]+\/approve/', $path) &&
+    !preg_match('/token=/', $path)) {
```

Comment:
Weird wrapping here; the conditionals should all be in a single line.

Reply:
Conditionals can break onto more than one line if the single line is more than 80 chars. However, we prefer creating variables for each condition, and keeping the conditional both on one line AND under 80 chars :)

Something like...
```php
$comment_approval_path = preg_match('/comment\/[1-9]+\/approve/', $path);
$token_path = preg_match('/token=/', $path);
if ($comment_approval_path && !$token_path) {
```